carbon-design-system/carbon-addons-iot-react
# fix(DashboardEditor): add correct labels

Closes #

**Summary**

- If a data item had no label, the field was appearing empty rather than showing the underlying id
- In the data series item modal, the `dataSourceId` was displaying instead of the true `dataItemId`

**Change List (commits, features, bugs, etc)**

- Moved the `columns` of the table to its own variable to keep the JSX cleaner for readability
- Moved the data list items to its own variable to keep the JSX cleaner for readability

**Acceptance Test (how to verify the PR)**

- Go to any DashboardEditor story, add a barchartcard, add a data item, edit the data item, remove the 'Custom label', close the modal, and see that the data series list item shows the dataSourceId rather than an empty label
- Go to any DashboardEditor story, add a barchartcard, add a data item, edit the data item, remove the 'Custom label', close the modal, and see that the title and labels show the underlying dataItemId rather than the dataSourceId
